web/src/features/AppVersionHistory/AppVersionHistoryRow.jsx

```diff
@@ -34,33 +36,6 @@ class AppVersionHistoryRow extends Component {
     );
   };
 
-  renderYamlErrors = (version) => {
```

Comment:
the new component doesn't take version- i flattened it out and passed in a callback.